coffee-backend/src/main/kotlin/chat/letscoffee/friends/FriendRequestModel.kt

```diff
@@ -0,0 +1,23 @@
+package chat.letscoffee.friends
+
+import chat.letscoffee.security.model.User
+import javax.persistence.*
+
+
+@Entity
+@Table(name = "friendrequest")
```

Comment:
Rename to `friend_request`, as it is normally written in two words, keeping word splitting consistently in tables